### ISSUE TITLE
Anyone with Software Carpentry?

### ISSUE THREAD

**Issue by hgibling** [UofTCoders/studyGroup#49]
I meant to ask at the Hacky Hour a couple weeks ago--is anyone else involved with Software Carpentry? I'm a new instructor and a couple of things I'm hoping to gain out of involvement with this study group is getting more comfortable with teaching and joining a network of other instructors.

If you haven't heard of [Software Carpentry](http://software-carpentry.org/), it's a great organization that runs free/low-cost [workshops](http://software-carpentry.org/workshops/index.html) around the world to improve the computer skills of researchers. They have all of their [lesson plans](http://software-carpentry.org/lessons.html) available online and cover [git](http://swcarpentry.github.io/git-novice/), [R](http://swcarpentry.github.io/r-novice-inflammation/), [python](http://swcarpentry.github.io/python-novice-inflammation/), [SQL](http://swcarpentry.github.io/sql-novice-survey/), [unix shell](http://swcarpentry.github.io/shell-novice/)... a great resource.

**Comment by QuLogic** [comment]
I think @lwjohnst86 is an instructor, too.

**Comment by lwjohnst86** [comment]
I am also an instructor with software carpentry! This is a good place to teach and learn! Welcome, it's a great place to share skills 😊

On 27 September, 2015 5:08:08 PM EDT, Heather Gibling notifications@github.com wrote:

> I meant to ask at the Hacky Hour a couple weeks ago--is anyone else
> involved with Software Carpentry? I'm a new instructor and a couple of
> things I'm hoping to gain out of involvement with this study group is
> getting more comfortable with teaching and joining a network of other
> instructors.
> 
> If you haven't heard of [Software
> Carpentry](http://software-carpentry.org/), it's a great organization
> that runs free/low-cost
> [workshops](http://software-carpentry.org/workshops/index.html) around
> the world to improve the computer skills of researchers. They have all
> of their [lesson plans](http://software-carpentry.org/lessons.html)
> available online and cover
> [git](http://swcarpentry.github.io/git-novice/),
> [R](http://swcarpentry.github.io/r-novice-inflammation/),
> [python](http://swcarpentry.github.io/python-novice-inflammation/),
> [SQL](http://swcarpentry.github.io/sql-novice-survey/), [unix
> shell](http://swcarpentry.github.io/shell-novice/)... a great resource.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/UofTCoders/studyGroup/issues/49

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by mbonsma** [comment]
@hgibling - is there anything in particular you're interested in teaching? There's some flex in the [schedule](https://github.com/UofTCoders/studyGroup/wiki/Schedule---Fall-2015) for this term (particularly in the python lightning demos and advanced R topic session), but if not this term we would love to have to lead a session next term.

**Comment by katieannemills** [comment]
@hgibling - if you're interested in getting to know the SWC network of instructors, have you heard about the [Instructor & Helper retreat](http://swcarpentry.github.io/instructor-retreat-2015/) I'm helping pull together (unrelated to Study Group)? There isn't yet a Toronto node, but I'd love some help pulling one together. If you can help lead on that, open a new issue [over here](https://github.com/swcarpentry/instructor-retreat-2015/issues). Thanks for getting involved in both projects!

**Comment by hgibling** [comment]
Thanks all!
@lwjohnst86 - awesome, I've already met you!
@mbonsma - I'd be most comfortable with R, but not until next semester, after I've finished rotations and am more settled at UofT.
@BillMills - I knew about the retreat and noticed there wasn't one for Toronto, but I'm new to Toronto and pretty swamped this semester. I'll keep an eye out though and offer my assistance if someone else takes the lead.

**Comment by katieannemills** [comment]
Hey folks,

@timtomch has stepped up to lead the Software Carpentry Instructor & Helper Retreat node for Toronto - this is a great chance for everyone who wants to find out more about that program to meet the community in Toronto and get involved. Software Carpentry and Study Groups are a great match, so I would encourage everyone to find out more; a SWC workshop is a great way to assemble a whole bunch of new learners, and Study Group is a great way to keep them practicing and getting together long term.

The details for the Toronto location are at [this etherpad](https://etherpad.wikimedia.org/p/swc-instructor-retreat-2015-toronto), but the site is a little up in the air still. It doesn't look like Mozilla is going to work for this one (no staff on the ground in Toronto at that time). Could Study Group help find a space somewhere?

**Comment by timtomch** [comment]
Thanks @BillMills! Yes do not hesitate to join us for the SWC instructor retreat on November 14, just sign up on the etherpad and we'll keep you in the loop.

We have a few other options open for rooms beside Mozilla, including UofT, I will let you know if we need help securing a location.

**Comment by hgibling** [comment]
Are etherpads back up and running? Some still seem to be down (SWC sign-ups for instructors, for example).

**Comment by timtomch** [comment]
@hgibling The etherpads that SWC was using used to be hosted by Mozilla, which apparently had to pull the plug for security reasons. This happened quite fast and it's possible the old etherpads are still being referenced all over our website.

Can you elaborate on the one you were trying to access? Thanks!

**Comment by hgibling** [comment]
@timtomch Ahh, didn't realize there were etherpads outside of Mozilla.
None on the website (that I've come across), just the ones normally emailed out for instructor recruitment. But no worries about them :)

**Comment by mbonsma** [comment]
Jon and I have been in contact about location for the retreat. Thanks for sharing this info @BillMills!

**Comment by pipitone** [comment]
Hey all, I'm the Jon @mbonsma was referring to, and I'm helping find space for the SWC retreat. I've already booked a space where I work, CAMH, but I'm still trying to find us a better spot either at Mozilla or U of T.

**Comment by katieannemills** [comment]
SWC etherpad update: yes, a lot of the pads are still stuck in limbo unfortunately. Help requests are open at Mozilla to get this information extracted, and teams are working on those presently, will hopefully have them cleared up soon. All SWC etherpads will be restored ASAP, albeit at new URLs; have a go at etherpad.wikimedia.org if you need to make one right away.

Looking forward to the Retreat, let me know any way I can help!
